gravitee-apim-console-webui/src/management/api/creation/v4/api-creation-v4.component.ts

```diff
@@ -158,8 +167,8 @@ export class ApiCreationV4Component implements OnInit, OnDestroy {
         } else {
           this.snackBarService.success(`API ${this.currentStep.payload.deploy ? 'deployed' : 'created'} successfully!`);
         }
-        if (result.result?.apiId) {
-          this.ajsState.go('management.apis.create-v4-confirmation', { apiId: result.result.apiId });
+        if (result.result?.api?.id) {
+          this.ajsState.go('management.apis.create-v4-confirmation', { apiId: result.result.api?.id });
```

Comment:
Is the check `result.result.api?.id` necessary with the `if (result.result?.api?.id)` above?